libs/libc/string/lib_strerror.c

```diff
@@ -367,6 +371,9 @@ static const struct errno_strmap_s g_errnomap[] =
 
 FAR char *strerror(int errnum)
 {
+#ifdef CONFIG_LIBC_STRERROR_ERRNUM
```

Comment:
I'm not sure that is worth of config option. Maybe just should be added unconditionally. @xiaoxiang781216 @davids5 @hartmannathan what do you think?

Reply:
Why this was marked as resolved without answer?

Reply:
I am fine with either(I guess others has the same feeling). If you prefer to enable it unconditionally, let's do it, thanks.